packages/core/README.md

```diff
@@ -465,6 +465,8 @@ function sendRequest(host, cb) {
 
 ### Capture all outgoing AWS requests
 
+This is only available for AWS SDK v2 due to the service-oriented architecture of AWS SDK v3.
```

Comment:
Probably good to add a link to the announcement about the difference between v2 and v3

Reply:
Good idea, just added